### PR TITLE
[Proposal] Refactor error handler and implement package for status codes

### PR DESCRIPTION
**Pull request by emanuelcasco**
## Summary

- Refactor in error handler and implementation of package for http status codes.
  + With this change you only have to add new errors in just one place (./app/errors.ts).

## Known Issues

- CI still broken

## Trello Card

N/A

### CODE REVIEW

File: app/controllers/users.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { NextFunction, Request, Response } from 'express';
+import HttpStatus from 'http-status-codes';
+
 import userService from '../services/users';
 import { User } from '../models/user';
-import { statusCodes } from './commons';
 import { notFound } from '../errors';
 
-export function getUsers(_: Request, res: Response, next: NextFunction): Promise<Response | void> {
+export function getUsers(req: Request, res: Response, next: NextFunction): Promise<Response | void> {
```
Comment (mpicollo-wolox) [inline app/controllers/users.ts]: If `req` is unused, `_` is not forced by linter ?

Comment (emanuelcasco) [reply app/controllers/users.ts]: no, it isn't. Besides, im not a big fan of using underscore for unused vars in js. For example, if you have more than one you have to name the other one.

**Comment by emanuelcasco** [comment]
@mpicollo-wolox I totally agree with the idea of wrapping http status codes info in a utils. I will change it!

File: app/controllers/users.ts (hunk 1 above)
Comment (mpicollo-wolox) [reply app/controllers/users.ts]: Yeah more than one is awful, it was only a question because the linter of JS does that

**Pull request merged**